modules/core/src/test/java/org/apache/ignite/internal/processors/security/client/AttributeSecurityCheckTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
```

Comment:
Please update copyright year in your IDE and all new classes.

Reply:
Done